### PR TITLE
AVM-Review-PR

### PR DESCRIPTION
**Pull request by xhy8759**
## Type of Change

- [x] Non-module change (e.g. CI/CD, documentation, etc.)
- [ ] Azure Verified Module updates:
  - [ ] Bugfix containing backwards compatible bug fixes, and I have NOT bumped the MAJOR or MINOR version in `locals.version.tf.json`:
    - [ ] Someone has opened a bug report issue, and I have included "Closes #{bug_report_issue_number}" in the PR description.
    - [ ] The bug was found by the module author, and no one has opened an issue to report it yet.
  - [ ] Feature update backwards compatible feature updates, and I have bumped the MINOR version in `locals.version.tf.json`.
  - [ ] Breaking changes and I have bumped the MAJOR version in `locals.version.tf.json`.
  - [ ] Update to documentation

# Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [x] My corresponding pipelines / checks run clean and green without any errors or warnings
- [x] I did run all  [pre-commit](https://azure.github.io/Azure-Verified-Modules/contributing/terraform/terraform-contribution-flow/#5-run-pre-commit-checks) checks

### CODE REVIEW

**Review by duzitong — APPROVED** [review]

**Review by mbilalamjad — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,23 +1,7 @@
 <!-- BEGIN_TF_DOCS -->
-# terraform-azurerm-avm-template
+# terraform-azapi-avm-ptn-site-manager
```
Comment (mbilalamjad) [inline README.md:2]: Please correct the name ot terraform-azurerm-avm-res-edge-site

File: README.md [hunk 2]
```diff
@@ -1,23 +1,7 @@
...
-Things to do:
-
-1. Set up a GitHub repo environment called `test`.
-1. Configure environment protection rule to ensure that approval is required before deploying to this environment.
-1. Create a user-assigned managed identity in your test subscription.
-1. Create a role assignment for the managed identity on your test subscription, use the minimum required role.
-1. Configure federated identity credentials on the user assigned managed identity. Use the GitHub environment.
-1. Search and update TODOs within the code and remove the TODO comments once complete.
-
-> [!IMPORTANT]
```
Comment (mbilalamjad) [inline README.md:15]: Please do not remove this IMPORTANT notice

File: _header.md [hunk 3]
```diff
@@ -1,19 +1,3 @@
-# terraform-azurerm-avm-template
+# terraform-azapi-avm-ptn-site-manager
```
Comment (mbilalamjad) [inline _header.md:1]: Please correct the name ot terraform-azurerm-avm-res-edge-site

File: _header.md [hunk 4]
```diff
@@ -1,19 +1,3 @@
...
-Things to do:
-
-1. Set up a GitHub repo environment called `test`.
-1. Configure environment protection rule to ensure that approval is required before deploying to this environment.
-1. Create a user-assigned managed identity in your test subscription.
-1. Create a role assignment for the managed identity on your test subscription, use the minimum required role.
-1. Configure federated identity credentials on the user assigned managed identity. Use the GitHub environment.
-1. Search and update TODOs within the code and remove the TODO comments once complete.
-
-> [!IMPORTANT]
```
Comment (mbilalamjad) [inline _header.md:14]: Please do not remove this IMPORTANT notice

**Review by mbilalamjad — CHANGES_REQUESTED** [review]
Please address the comments

**Review by mbilalamjad — APPROVED** [review]

**Pull request merged**